product-service/api-documentation.yaml

```diff
@@ -0,0 +1,76 @@
+openapi: 3.1.0
+info:
+  title: Leaf shop API
+  description: |-
+    Docs for the Leaf shop API. I'm only doing it for 4 points honestly.
+  version: 1.0.0
+servers:
+  - url: https://9b9no1dw79.execute-api.eu-west-1.amazonaws.com
+tags:
+  - name: products
+    description: Working with products
+paths:
+  /products:
+    get:
+      tags:
+        - products
+      summary: Get the list of products
+      description: Get the list of products
+      operationId: getProducts
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Products'
+        '400':
+          description: Random error
+  /products/{id}:
+    get:
+      tags:
+        - products
```

Comment:
seems that tags, summary and description were copy pasted from products API. I assume this supposed to be auto generated, you can use serverless-openapi plugin and set all needed fields as in this doc https://www.serverless.com/plugins/serverless-openapi-documentation it can save your time in the future